src/ralph/static/src/js/ralph.init.js

```diff
@@ -1,2 +1,74 @@
 var ralph = ralph || {};
 ralph.jQuery = jQuery.noConflict();
+
+function getCallerNode(win) {
```

Comment:
why this have to be in `ralph.init.js`? couldn't it be moved to some autocomplete-related file?